tools/downloader/src/open_model_zoo/model_tools/_configuration.py

```diff
@@ -435,8 +461,12 @@ def load_models_from_args(parser, args):
         models = collections.OrderedDict() # deduplicate models while preserving order
 
         for pattern in patterns:
-            matching_models = [model for model in all_models
-                if fnmatch.fnmatchcase(model.name, pattern)]
+            matching_models = []
+            for model in all_models:
+                if fnmatch.fnmatchcase(model.name, pattern):
+                    matching_models.append(model)
+                elif model.composite_model_name and fnmatch.fnmatchcase(model.composite_model_name, pattern):
```

Comment:
The documentation for the `--name` and `--list` options should be updated to note that both model names and composite model names are accepted.

Reply:
Done